packages/api/methods.ts

```diff
@@ -192,3 +192,27 @@ export function updateRule(rule) {
 export function deleteRule(id) {
   return send('api/rule-delete', { id });
 }
+
+export function getSchedules() {
+  return send('api/schedules-get');
+}
+
+export function createSchedule(schedule, conditions) {
+  return send('api/schedule-create', { schedule, conditions });
+}
+
+export function updateSchedule(schedule, conditions, resetNextDate) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think we should follow the same pattern used for other API methods here. First argument is the `id` of the resource that will be updated. The next arguments are the "what" - the things that actually get updated.